third_party/amd/lib/TritonAMDGPUTransforms/AccelerateAMDMatmul.cpp

```diff
@@ -659,21 +707,13 @@ class TritonAMDGPUAccelerateMatmulPass
     MLIRContext *context = &getContext();
     ModuleOp m = getOperation();
 
-    RewritePatternSet patterns(context);
-    switch (auto isaFamily = triton::AMD::deduceISAFamily(archGenerationName)) {
```

Comment:
Can we keep the original way of reusing `deduceISAFamily` and have a new function for `getWmmaVersion` to take in the arch string directly?